src/components/footer/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <div className="footer">
+      <div>
+        <p> Meet the team: </p>
+      </div>
+
+      <div className="teamMembers">
+        <p>
+          Alejandro Patino, Daria Fetko, Olga Musteata, Elena Demidenko, Alzhan
```

Comment:
each team member should be wrapped in their own p tags so we can add linkedIn and GH icons for each person that will lead to individual linkedIn and GH pages.

If you have time to add those icons, that would be awesome - if not, no problem.